### PR TITLE
Updated year in Xcode's file template 

### PR DESCRIPTION
**Pull request by SPaluchiewiczSofomo**

### CODE REVIEW

File: ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <dict>
 	<key>FILEHEADER</key>
 	<string>
-//  Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+//  Copyright (c) 2020 Magic Leap, Inc. All Rights Reserved
```
Comment (pleszkiewicz) [inline ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist]: Should we include 2019 as well? -> `2019-2020`

Comment (SPaluchiewiczSofomo) [reply ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist]: According to information from @kpiascik we should set this to `2019-2020` but we're waiting for final confirmation.

Comment (kpiascik) [reply ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist]: Please use this:
`// Copyright (c) 2019-2020 Magic Leap, Inc. All Rights Reserved.`

Comment (SPaluchiewiczSofomo) [reply ios/RNMagicScript.xcodeproj/xcshareddata/IDETemplateMacros.plist]: Find & replace for current files will be provided in separate PR.

**Pull request merged**